### PR TITLE
[SQL] Document how to create a timestamp from Unix epoch

### PR DESCRIPTION
**Pull request by mihaibudiu**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -11,7 +11,7 @@ members = [
   "sql-to-dbsp-compiler/lib/sqllib",
 ]
 exclude = [
-  "sql-to-dbsp-compiler/temp",
+  "sql-to-dbsp-compiler",
```
Comment (ryzhyk) [inline Cargo.toml:14]: Is this change intentional?

File: docs/sql/datetime.md [hunk 2]
```diff
@@ -213,6 +213,14 @@ result is as follows:
 - Adding a interval of hours, minutes, or seconds to a `DATE` produces a `TIMESTAMP` result
 - Adding a an interval of days, months, or longer to a `DATE` produces a `DATE` result
 
+To create a timestamp using the Unix EPOCH as a base, you can use the
```
Comment (ryzhyk) [inline docs/sql/datetime.md:216]:
```suggestion
To create a timestamp using the Unix EPOCH in seconds as a base, you can use the `TIMESTAMPADD` function:
```

**Review by ryzhyk — APPROVED** [review]

File: Cargo.toml (hunk 1 above)
Comment (mihaibudiu) [reply Cargo.toml:14]: yes, I am adding some more directories with experiments
